### PR TITLE
Add details about Softwarepraktikum 2021

### PR DESCRIPTION
**Pull request by jaimergp**
This adds:
* [x] Blog post
* [x] OpenCADD project page that was missing (!)
* [X] PDF of last year for archival
* [x] Slides for this year (WIP)

### CODE REVIEW

File: content/blog/structural-alignment-2021.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+The new edition of the "Structural Alignment and Superposition" internship is now open for registration!
+
+In this internship, an open-source structural alignment Python package for biopolymers, i.e. proteins and ligands, that we started implementing in 2020 will be further developed and released. While diverse structural alignment implementations can be found in visualization software such as PyMol, VMD or UCSF Chimera, a standalone package is currently missing in the Python ecosystem. We have started filling that gap with a modern Python package designed under the current best practices for development, testing and deployment.
+
+The main focus of this year’s course will be on finalizing the package. This will include, getting familiar with the current code base, working on alignment-specific issues such as automatic structural renumbering as well as benchmarking the tool.
+
+More details:
+
+- Check the [slides](#missing).
```
Comment (AndreaVolkamer) [inline content/blog/structural-alignment-2021.md:14]: Todo add link once finished

File: content/education/cadd-seminar.md [hunk 2]
```diff
@@ -3,24 +3,24 @@ title: CADD Courses & Seminars @ FU
...
 
-* Seminar (60101711): Computer-Aided Drug Design - Methods and Application - <a href="https://mycampus.imp.fu-berlin.de/evento/vv/course/597249?locale=en" target="_blank" class="external">SS2020</a>
-* Software and project management (60102613): <a href="https://www.mi.fu-berlin.de/bioinf/stud/bachelor/abv/projektmanagement/SWP_projekte/Structural_Alignment.html" target="_blank" class="external">Structural Alignment - SS2020</a>
-* Practical Course (60101813): Using open source software and libraries for data analysis and computer-aided drug design - <a href="https://mycampus.imp.fu-berlin.de/evento/vv/course/546987" target="_blank" class="external">WS19/20</a>
+- Seminar (60101711): Computer-Aided Drug Design - Methods and Application - <a href="https://mycampus.imp.fu-berlin.de/evento/vv/course/597249?locale=en" target="_blank" class="external">SS2020</a>
+- Software and project management (60102613): <a href="https://www.mi.fu-berlin.de/bioinf/stud/bachelor/abv/projektmanagement/SWP_projekte/Structural_Alignment.html" target="_blank" class="external">Structural Alignment with OpenCADD</a>. [More details](/projects/opencadd/).
```
Comment (AndreaVolkamer) [inline content/education/cadd-seminar.md:20]:
```suggestion
- Software and project management (60102613): <a href="https://www.mi.fu-berlin.de/bioinf/stud/bachelor/abv/projektmanagement/SWP_projekte/Structural_Alignment.html" target="_blank" class="external">Structural Alignment with OpenCADD WS20/21</a> (continued from WS19/20). [More details about OpenCADD](/projects/opencadd/).
```

File: content/blog/structural-alignment-2021.md (hunk 1 above)
Comment (AndreaVolkamer) [reply content/blog/structural-alignment-2021.md:14]: pdf from 2020 was added to static/files, but not the updated one?

**Review by AndreaVolkamer — APPROVED** [review]
@jaimergp thanks, well done! Small comments added (note the pdf that is currently added to static/files is the one from 2020), can be merged after inclusion without further approval.

File: content/blog/structural-alignment-2021.md (hunk 1 above)
Comment (jaimergp) [reply content/blog/structural-alignment-2021.md:14]: Yes, will upload once we are happy with it!

**Pull request merged**